backend/wordAlgorithm/word.py

```diff
@@ -0,0 +1,94 @@
+# Format of xml taken as input:
+#    <Words>
+#        <Word>
+#            <Language>Dutch</Language>
+#            <Word>ja</Word>
+#            <Translation>yes</Translation>
+#            <Category/>
+#            <Word_Difficulty/>
+#            <Extensions/>
+#        </Word>
+#    </Words>
+
+
+words = []
+
+# Word object
+class Word(object):
+    originalWord = "";
+    language = ""
+    translatedWord = "";
+    category = "";
+    wordValue = 0;
+    extensions = []
+    progress = 0;
+    wordDifficulty = 0;
+    
+# Word constructor
+def makeWord(language, originalWord, translatedWord, category, extensions, wordDifficulty):
```

Comment:
IMPORTANT: Just because you can comment everything doesn't mean you should. The very simple functions are not expected to have comments ALTHOUGH i would like to at least contain what type the variables/parameter are (see comment on previous file)